metrics/proxy.go

```diff
@@ -21,6 +21,7 @@ func MetricedFullAPI(a api.FullNode) api.FullNode {
 	var out apistruct.FullNodeStruct
 	proxy(a, &out.Internal)
 	proxy(a, &out.CommonStruct.Internal)
+	//proxy(a, &out.HelloWorldStruct.Internal)
```

Comment:
This line is required in order for the hello world dependency to work. Without it, the client will panic when calling the `Hello` method. Smells a bit funky :cheese:

Reply:
@Kubuxu Looks like this is where part of the problem lies.

Reply:
Best guess is that something may be amiss here: https://github.com/filecoin-project/go-jsonrpc/blob/master/client.go#L284-L305